disnake/__init__.py

```diff
@@ -72,6 +72,6 @@ class VersionInfo(NamedTuple):
     serial: int
 
 
-version_info: VersionInfo = VersionInfo(major=2, minor=3, micro=2, releaselevel="beta", serial=0)
+version_info: VersionInfo = VersionInfo(major=2, minor=4, micro=0, releaselevel="beta", serial=0)
```

Comment:
should we consider changing the release level to stable here, we are no longer a beta product.